pkg/rbd/rbd.go

```diff
@@ -110,6 +110,7 @@ func (r *Driver) Run(driverName, nodeID, endpoint, configRoot string, containeri
 		csi.ControllerServiceCapability_RPC_CREATE_DELETE_SNAPSHOT,
 		csi.ControllerServiceCapability_RPC_LIST_SNAPSHOTS,
 		csi.ControllerServiceCapability_RPC_CLONE_VOLUME,
+		csi.ControllerServiceCapability_RPC_EXPAND_VOLUME,
```

Comment:
@humblec I did not understand why we are adding this capability when we actually do not have it?